__test__/integrationTests.test.ts

```diff
@@ -252,3 +281,17 @@ describe("Integration tests for AUTH routes:", () => {
         });
     });
 });
+
+interface UserRegister {
```

Comment:
An identical interface already exists at ./models/userModel.ts called RegistrationData. You can import it and use it directly.

Reply:
I've imported it from userModel.